client/lib/js/utils/request.js

```diff
@@ -0,0 +1,38 @@
+import { request as baseRequest, sanitize } from 'js-utility-belt/es6';
+
+import ApiUrls from '../constants/api_urls';
+
+
+const DEFAULT_REQUEST_CONFIG = {
+    credentials: 'include',
+    headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+    }
+};
+
+/**
+ * Small wrapper around js-utility-belt's request that provides default settings and response
+ * handling
+ */
+export default function request(url, config) {
```

Comment:
ok

Op di 31 mei 2016 om 14:15 schreef Brett Sun <notifications@github.com>:

> In client/lib/js/utils/request.js
> <https://github.com/bigchaindb/bigchaindb-examples/pull/14#discussion_r65169970>
> :
>
> > +import ApiUrls from '../constants/api_urls';
> > +
> > +
> > +const DEFAULT_REQUEST_CONFIG = {
> > +    credentials: 'include',
> > +    headers: {
> > +        'Accept': 'application/json',
> > +        'Content-Type': 'application/json'
> > +    }
> > +};
> > +
> > +/**
> > + * Small wrapper around js-utility-belt's request that provides default settings and response
> > + * handling
> > + */
> > +export default function request(url, config) {
>
> I'd shy away from it since it really loses flexibility very quickly if we
> do things like assuming json responses and stripping null query parameters.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bigchaindb/bigchaindb-examples/pull/14/files/9dc1a83dc43ea93dc2cec32b74a98668f4be5603#r65169970>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF5HJW-nzVoiNHFuBuJsrAreLkZAPZAWks5qHCZagaJpZM4IqU-J>
> .
>